src/frontend/komponenter/Infotrygd/Infotrygd.tsx

```diff
@@ -44,10 +44,11 @@ export const Infotrygd: React.FC = () => {
     const { ident, onSubmitWrapper, skjema } = useInfotrygdSkjema();
     const { flyttBrukerTilBaSak, migrerInfotrygdSakRessurs } = useInfotrygdMigrering();
 
-    const history = useHistory<{ bruker: string } | undefined>();
+    const history = useLocation();
     useEffect(() => {
-        if (history.location.state) {
-            skjema.felter.ident.verdi = history.location.state.bruker;
+        if (history.state) {
+            const state = history.state as { bruker: string };
+            skjema.felter.ident.verdi = state.bruker;
```

Comment:
Er ikke så fan av å bruke `as` for det ødelegger jo egentlig typingen, men så det var gjort sånn mange steder på nett også 🤷‍♂️
Ville
```
        const bruker = history.state?.bruker
        if (history.state) {
            skjema.felter.ident.verdi = bruker;
            onSubmitWrapper();
```
Blitt enklere?

Synes også det er rart at `skjema.felter.ident.verdi` muteres direkte. Ser det er det som er gjort tidligere, men trodde egentlig man skulle bruke `skjema.felter.ident.validerOgSettFelt()` 🤔

Reply:
Vil då få inspector-feil på const bruker = history.state?.bruker. history.state er alltid any, så må nesten bruke as tror eg.